src/System.Private.Xml/src/System/Xml/Serialization/ReflectionXmlSerializationWriter.cs

```diff
@@ -951,7 +954,7 @@ private void WritePrimitive(WritePrimitiveMethodRequirement method, string name,
                 {
                     if (((EnumMapping)mapping).IsFlags)
                     {
-                        var defaultEnumFlagValues = defaultValue.ToString().Split(null).Where((s) => !string.IsNullOrWhiteSpace(s));
+                        IEnumerable<string> defaultEnumFlagValues = defaultValue.ToString().Split(null).Where((s) => !string.IsNullOrWhiteSpace(s));
```

Comment:
>.Where((s) => !string.IsNullOrWhiteSpace(s)); [](start = 103, length = 45)

What's this Where clause for? Passing null to Split means it will split the string on whitespace. If there are two whitespace characters in a row, then the Split() will emit a null string in between the two white space characters. It looks like the Where clause is removing any useless nulls which can be done more cheaply using StringSplitOptions.RemoveEmptyEntries when calling Split.